### ISSUE TITLE
FMS does not compile topography.F90 with GCC 11

### ISSUE THREAD

**Issue by marshallward** [NOAA-GFDL/FMS#753]
**Describe the bug**
Compiling topography.F90 with GCC 11 produces this error:
```
topography.F90:104:63:

  104 |    namelist /topography_nml/ topog_file, water_file, use_mpp_io
      |                                                               1
Error: Symbol 'use_mpp_io' in namelist 'topography_nml' at (1) must be declared before the namelist is declared.
```

**To Reproduce**
Use the GCC compiler and follow the autoconf build.

**System Environment**
Describe the system environment, include:
- OS:
```
$ uname -a
Linux [...] 5.12.5-arch1-1 #1 SMP PREEMPT Wed, 19 May 2021 10:32:40 +0000 x86_64 GNU/Linux
```
- Compiler(s): 
```
$ gfortran --version
GNU Fortran (GCC) 11.1.0
```

**Other**
This was first detected in 2020.04.

**Comment by thomas-robinson** [comment]
@marshallward I can see this going in as a 2021.02.01 patch.  I don't know if we will patch older versions.  Do you *NEED* it in 2020.04?

**Comment by marshallward** [comment]
No I think we can live without it.

**Comment by thomas-robinson** [comment]
OK we can consult with the team and decide on a patch for 2021.02.

**Comment by rem1776** [comment]
Due to the other issues/bugs with GCC 11, we'll be including fixes for support in the next release rather than creating a new patched version.

**Comment by rem1776** [comment]
Fixed with mpp_io removal iin #760
